src/mumble/MainWindow.cpp

```diff
@@ -333,7 +333,7 @@ void MainWindow::setupGui()  {
 
 	updateTransmitModeComboBox();
 
-#if QT_VERSION < 0x050000 && !defined(Q_OS_MAC)
+#if defined(Q_OS_WIN) || (QT_VERSION < 0x050000 && !defined(Q_OS_MAC))
```

Comment:
I guess that this should technically be `QT_VERSION >= 0x050000 && defined(Q_OS_WIN)`?

How about:

    // For Qt >= 5, enable this call for Windows.
    // For Qt < 5, enable for anything but macOS.
    #if (QT_VERSION >= 0x050000 && defined(Q_OS_WIN) || (QT_VERSION < 0x050000 && !defined(Q_OS_MAC))

?

Only a technicality... I don't believe we actually support Qt 4 on Windows for 1.3.x.